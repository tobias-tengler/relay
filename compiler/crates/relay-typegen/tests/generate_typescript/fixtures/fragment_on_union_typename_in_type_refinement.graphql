fragment c on MaybeNode {
  ... on NonNode {
    name
  }
  ... on Story {
    __typename
    actorCount
  }
}