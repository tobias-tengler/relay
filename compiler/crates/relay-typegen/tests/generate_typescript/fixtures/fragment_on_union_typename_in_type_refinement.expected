==================================== INPUT ====================================
fragment c on MaybeNode {
  ... on NonNode {
    name
  }
  ... on Story {
    __typename
    actorCount
  }
}
==================================== OUTPUT ===================================
import { FragmentRefs } from "relay-runtime";
export type c$data = {
  readonly __typename: "Story";
  readonly actorCount?: number | null | undefined;
  readonly name?: string | null | undefined;
  readonly " $fragmentType": "c";
};
export type c$key = {
  readonly " $data"?: c$data;
  readonly " $fragmentSpreads": FragmentRefs<"c">;
};
